transports/quic/Cargo.toml

```diff
@@ -28,6 +28,7 @@ ring = { workspace = true }
 [features]
 tokio = ["dep:tokio", "if-watch/tokio", "quinn/runtime-tokio"]
 async-std = ["dep:async-std", "if-watch/smol", "quinn/runtime-async-std"]
+default = ["tokio"]
```

Comment:
We probably should not have it default to tokio right now since it would require disabling default features to enable the needed feature